src/applications/gi-sandbox/components/profile/CalculateYourBenefitsForm.jsx

```diff
@@ -842,7 +843,7 @@ function CalculateYourBenefitsForm({
             "I'll be taking classes outside of the U.S. and U.S. territories"
           }
           onChange={handleHasClassesOutsideUSChange}
-          checked={inputs.classesOutsideUS}
+          checked={classesOutsideUSChecked}
           name={'classesOutsideUS'}
```

Comment:
Sorry to butt in but the reason the checkbox doesn't check is because the `Checkbox` component turns this name prop into `classesoutsideus`, so the `field` value (e.target.name) when calling `calculatorInputChange` above is actually `classesoutsideus`, not `classesOutsideUS` like you would expect. Because of this `inputs.classesOutsideUS` is always false, and the checkbox remains unchecked

Reply:
Ah. Interesting quirk. I'll make the change so an extra property won't be required.

Reply:
You will want to update your reducer (reducers/calculator.js) if you want to change the property. Another option is to update the `handleHasClassesOutsideUSChange` to call `handleCheckboxChange('classesOutsideUS', e.target.value)` and update `handleCheckboxChange` to accept a field and value as parameters instead of an event.

Reply:
I just changed the checked property to inputs.classesoutsideus & the earlier reference to control zip code input presentation to the same lowercased property and it seems to work correctly.

Updated screenshots show it affects presentation as well too (neglected this before as it wasn't in the description of expected behavior but this behavior looks correct).

Reply:
You might want to make sure that the selectors (selectors/calculator.js) still work as well